simplq/src/components/pages/Status/LeaveQueue.jsx

```diff
@@ -1,16 +1,23 @@
 import React from 'react';
+import CloseIcon from '@material-ui/icons/Close';
+import { useHistory } from 'react-router-dom';
 import styles from '../../../styles/statusPage.module.scss';
 
-export default () => {
+export default (props) => {
+  const history = useHistory();
+  function handleClick() {
+    props.leaveQueueHandler();
+    history.push(`/`);
+  }
+
```

Comment:
If I leave and then click the back button, I still get the old status page. @daltonfury42, could this be a backend issue?

Reply:
It's a front end thing. We can decide on this later.